src/app/onboarding/components/archive-type-select/archive-type-select.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+/* format */
```

Comment:
```suggestion
/* @format */
```
Another small issue with the format pragma.